app/routes/goal_routes.py

```diff
@@ -0,0 +1,42 @@
+from .helper_functions import create_instance, get_all_instances, get_one_instance, get_one_instance, delete_instance, update_instance,  add_tasks_to_model
```

Comment:
Great job separating out your helper functions! And great job with creating the helper functions themselves! It truly helps streamline the routes!